% !TEX TS-program = xelatex
\documentclass[12pt]{article}

\usepackage[margin=2cm]{geometry}

\usepackage{times}
\usepackage{lineno}
\usepackage[round]{natbib}
\makeatletter
\renewcommand{\@biblabel}[1]{#1.}
\makeatother

\usepackage{graphicx}
\usepackage{amsmath}

\usepackage{url}
\urlstyle{same}

\usepackage[small,compact]{titlesec}

%\titleformat{\section} {\vspace{24pt}\bf\sffamily\MakeUppercase}{\thesection} {0pt} {}
\titleformat{\section} {\vspace{12pt}\bf\large}{\thesection}{0pt}{}
\titleformat{\subsection} {\vspace{6pt}\bf}{\thesubsection} {0pt} {\vspace{-2pt}}
\titleformat{\subsubsection} [runin] {\bf}{\thesubsubsection} {12pt} {}

 
\begin{document}

\title{Blind dating: A phylogenetic approach to dating HIV reservoir sequences}

\author{Bradley R. Jones$^{1,2}$, Joshua Horacsek$^{1,2}$, Jeffrey B. Joy$^2$, \\ Zabrina L. Brumme$^{1,2}$, and Art F.Y. Poon$^{1,2,3}$}
%%% 1 Simon Fraser University
%%% 2 BC Centre for Excellence in HIV/AIDS
%%% 3 University of British Columbia
\baselineskip 22pt
\pagewiselinenumbers

\date{}
\maketitle

\section * {Abstract}

Background: 
The ability of HIV to persist within latent cellular reservoirs represents a major barrier to cure. The timing of establishment of individual viral reservoirs over the infection course may influence their susceptibility to elimination by immune-mediated or therapeutic approaches. However, methods to accurately estimate the age of reservoir sequences remain scarce. We propose a simple method to date suspected reservoir sequences using phylogenetically informed regression.

Method: 
Simulated sequence data were generated using INDELible version 1.03. A total of 410 HIV RNA sequences sampled longitudinally from 9 untreated HIV-infected individuals with estimated dates of infection and a total of 2474 HIV DNA and RNA sequences from 17 individuals were obtained from the Los Alamos National Laboratory HIV Sequence Database. The simulated and HIV RNA data were used for method validation. Maximum-likelihood phylogenies were reconstructed with RaxML. Phylogenies were rooted by determining the location of the root that minimized the root-mean-square error between root-to-tip distances and known dates of sampling. The root-to-tip distances of latent sequences were mapped to the optimal regression line to estimate reservoir establishment dates.

Results: 
To validate the method, we synthesized latent lineages by censoring the sample dates for random selections of up to 50\% of sequences in the simulated or HIV RNA datasets. In either case, root-to-tip estimation accurately recovered these missing sample dates when the phylogeny conformed to a strict molecular clock. When this method was applied to HIV DNA sequences in phylogenies containing dated HIV RNA sequences, we observed that the predicted dates tended to precede dates of sampling, which was consistent with latency.

Conclusions: 
Our results indicate that the dates that HIV DNA lineages entered a latent state can be reconstructed from a phylogenetic analysis of HIV RNA sequence variation from the same patient. This can provide important insights into the dynamics of HIV reservoirs within hosts.

Keywords: 
HIV, Latency, Cellular Reservoirs, Phylogenetics, Linear Regression\\



\underline{}
\section * {Introduction} \label{sec:intro}

A major obstacle along the path of developing HIV cure strategies is the persistence of the virus within latently infected cellular reservoirs \citep{Pace11}. 
Latently infected cells contain integrated HIV DNA and have entered a dormant state in which they not only have a lowered rate of virion production, but may also persist within an individual for several years or more.
Although a patient's viral load can be reduced below detectable levels when that patient is subjected to highly active antiretroviral therapy (HAART), latent reservoirs will eventually reseed an infection if a patient halts treatment \citep{Joos08, Pomerantz03, Richman09}. 
Not much is known about when specific latent reservoirs are established over the course of an infection. 
The chronology of their establishment is of interest as the distribution of ages of the latent virus lineages may provide information concerning the types of adaptations that those lineages may have accumulated. 
Specifically, the timing of reservoir establishment may influence how virions produced by those reservoirs will react to immune-mediated and/or therapeutic treatments. 
Previous studies have developed techniques to detect latent lineages by reduced rates of viral evolution \citep{Immonen14} and modelled the population dynamics of different classes of the virus within host \citep{Althaus14}. 
However, there has been limited work on estimating when viral lineages suspected of being latent became integrated into the host genome, or qualitatively assessing the distribution of collection-age differences for suspected latent samples.

An HIV-1 population rapidly accumulates genetic diversity over the course of an infection, due to its high mutation rate and short generation time %, and intense selective pressure by the host's immune system 
\citep{Alizon13, Shankarappa99, Rambaut04}. 
Using this extensive sequence diversity, it is possible to reconstruct a phylogenetic tree that represents the descent of virus lineages that have been sampled from a given infection from their common ancestors. 
A phylogeny that is reconstructed by parameterizing a model of molecular evolution \cite[\textit{e.g.}, by maximum likelihood estimation;][]{Felsenstein:1981} solely on the basis of sequence homology has branch lengths in units of expected numbers of substitutions, in which the passage of time is confounded with the rate of evolution.
In other words, a tree that stretches far back in time with a slow rate of evolution will explain a given set of sequences just as effectively as a shallow tree with rapid evolution.  
When lineages are known to be sampled at different times, however, the corresponding tips of the phylogeny can be fixed in time and the rate of evolution between the tips can be estimated directly \citep{Rodrigo99}.
This additional information enables one to rescale a phylogeny in chronological time, which is especially true for HIV-1 populations in which genetic differences can accumulate in weeks or months \citep{Williamson:2003}.
%With an appropriate method of relating evolutionary distance to time (a {\em clock model}), the phylogeny can be re-scaled chronologically, then by fixing the tips to their respective sampling dates, clock ``rates'' can be estimated .
Among the actively replicating HIV lineages within a host, the rate of molecular evolution is fairly consistent over time \citep{Leitner99, Kuhner95, Korber00}.
Consequently, the divergence of these lineages can be adequately approximated by a `strict' molecular clock model in which a single rate of evolution is sustained throughout.
%For sequences originating from the active population of virus, it is not unreasonable to assume that the rate of molecular evolution is relatively constant --- in other words, it adheres to a `molecular clock' model . 
Once a virus lineage becomes integrated into a host genome, however, its rate of evolution becomes negligible relative to the baseline rate in the actively replicating population.
Thus, viral latency effectively halts the molecular evolution of that particular virus lineage until reactivation. 

%Between the active and latent populations, latent cells are then expected to be more genetically different to active lineages than active lineages are among each other.

Viral latency can be manifested by a discordance between a sample's actual collection time and its apparent age based on its sequence divergence from other lineages (Figure \ref{fig:latenttree}). 
Therefore, it may be possible to use phylogenetic analysis to reconstruct the dates that a given virus lineage became latent.
We propose a simple framework to estimate dates of latent reservoirs which assumes that HIV evolution within a host can be adequately modelled by a strict molecular clock, such that the expected number of substitutions per site increases linearly with time \citep{Ho14}. 
%Our method extracts timing information from the phylogenetic relationships between the virus sampled at various time-points along the infection. 
First, we construct a phylogeny relating sequences derived from both cellular HIV DNA and plasma HIV RNA sampled from the same patient.
Next, we parameterize a strict clock by fitting a linear model to the known collection dates of the HIV RNA sequences and the path lengths from the corresponding tips of the phylogeny to the root, which represents the earliest point in time in the phylogeny. 
Since phylogenies reconstructed by the comparative analysis of sequences are typically unrooted, we employ both root-to-tip regression \citep{Korber00} and outgroup rooting methods to estimate the location of the root in the tree. 
We assume that sequences derived from cellular HIV DNA such as that extracted from peripheral blood mononuclear cells (PBMCs) are potentially derived from latent reservoirs. %, and therefore have effectively stopped evolving.
Although sequences derived from plasma HIV RNA may potentially be descended from reactivated latent virus lineages, we assume that this affects only a minority of sequences derived from blood plasma on the time scales represented by the longitudinal data sets in our study (see below).
We used simulated and real longitudinal HIV sequence data sets to evaluate the accuracy of reconstructing tip dates by this approach.
Finally, we applied our method to characterize the integration dates of viral lineages sampled as cellular HIV DNA.

%
%Next, we evaluated the same model on sequence data that was simulated under a model that included viral latency in the form of a variable molecular clock.
%To mimic the types of results we expected from our first test, we used a collection of published longitudinal patient-derived sequence data sets comprising exclusively plasma-derived HIV sequences \citep{McCloskey14} to evaluate the accuracy of date reconstruction for sequences with censored dates. 
%Finally, we tested our methodology on another collection of longitudinal patient-derived data sets comprising both PBMC and plasma sequences. 
%If the molecular clock assumption holds for all sequences, and latency merely implies a ``pause'' in the evolution of a sequence, then the clock should be a reliable source of information for dating latent sequences.




%%%%%%%%%%%%%%%%%%%%%%%% METHODS %%%%%%%%%%%%%%%%%%%%%%%%%%

\section * {Methods} \label{sec:methods}



\subsection * {Data Collection} \label{subsec:dcollection}
We collected two distinct sets of data for our experiments. 
The first we refer to as the {\em plasma} data set, which comprised over 19,000 published HIV \textit{env} sequences derived from viral RNA from longitudinal plasma samples ($n=335$) collected from 232 different individuals \citep{McCloskey14}. 
From these data, we selected treatment naive individuals with two or more clonal or single-genome sequences available at each time point, and with a known time-line relative to one of several reference points: HIV infection, seroconversion or presentation of symptomatic seroconversion illness. 
The baseline samples were collected within 186 days of this reference point, and at least one of the subsequent ``follow-up'' time points occurred a minimum of 6 months after baseline.
Individuals who had been classified as ``superinfected'' were also excluded \citep{McCloskey14}.
After applying these criteria, we retained 410 HIV RNA sequences from 9 individuals for phylogenetic analysis.
The purpose of the plasma data set was to provide representative samples of within-host HIV diversity for evaluating and calibrating the model.

The second data set, referred to as the {\em mixed} data set, contained sequences from studies in which both plasma HIV RNA and cellular HIV DNA had been collected from the same individual.
We used the Los Alamos advanced search interface \citep{LosAlamos} to identify published data sets representing individuals with both HIV RNA and cellular DNA sequences.
%% what do you mean by "well-characterized"?
From these results, we manually selected 16 individuals that had well-characterized partial {\em env} sequences and plasma samples from at least 2 RNA time points and 5 DNA time points. (Table \ref{tab:patients}). 
We also required that at least 1 DNA time point fell between 2 RNA time points in each indiviudal, other than this there were no selection criteria on the time between sample collection dates. 
A total of 926 partial {\em env} sequences from blood plasma, and 1548 partial {\em env} sequences from PBMC cells were used in our analysis. 
We utilized samples from treatment naive individuals from \cite{Shankarappa99, Novitsky09}, samples from individuals that had been treated via HAART from \cite{Llewellyn06}.
%, and samples from individuals that had been on HAART, but ceased treatment \cite{Fischer04}. 

\subsection * {Sequence Alignment} \label{subsec:seqalign}
When applicable, we used MUSCLE 3.8.31 \citep{Muscle04} to align the patient-derived sequences, which were then visually inspected and cleaned with AliView \citep{AliView14}. 
Alignments were manually trimmed to the interval in which at least  50\% of sequences within the data set had coverage over each base.
%Since the simulated sequence data contained no insertions or deletions, no alignment of these data was necessary. 
The plasma data set had already been aligned and cleaned \citep{McCloskey14}, so no further alignment was performed.

\subsection * {Simulated Data} \label{subsec:simdata}

%%% 1. simulate trees with strict clock - baseline
%%% 2. directly manipulate plasma trees
%%% 3. simulate sequence evolution on trees - additional error

To characterize sources of error in reconstructing dates on tips of a phylogeny, we generated phylogenetic trees by direct simulation. % and from the plasma data sets.
First, we generated phylogenies with 100 tips each under a birth-death model with serial sampling using the `sim.bdsky.stt' function in the \textit{R} package \textit{TreeSim} \citep{Boskova14}.
Birth-death models have been applied to studying speciation processes \citep{Nee:2006} and infectious disease epidemics \citep{Stradler13}, but have only recently been used to model the proliferation of virus lineages in cell populations \citep{Hartfield:2015}.
%% This paragraph needs to be rewritten
The birth-death model was parameterized by fitting this model to samples from the {\em plasma} dataset.
%% we may need to provide some justification for using BD
%To assess the sensitivity of our method, we first simulated HIV sequence data sets without viral latency.
We used BEAST version 2.3.2 to estimate the posterior distribution defined by the serial model with a Bayesian skyline, a strict molecular clock, and the HKY85 \citep{HKY85} model of nucleotide substitution.
%Using this approach, we generated samples of trees for each sequence alignment in the plasma dataset.
%From these simulations, we evaluated the accuracy of predicting sample collection times from a strict molecular clock model that was fit to a training subset of these data.
The lineage birth rate, death rate and sampling probability were estimated to be: $0.05116, 0.05006, 0.005237$ respectively.

We used the \textit{R} package, NELSI \citep{NELSI}, to assign rates of evolution to these trees by drawing values from a Gaussian distribution with a mean rate $\mu = \ 1.964\times 10^{-4}$ substitutions per generation and standard deviation $\sigma = \ 1.417\times 10^{-5}$. %; Figure \ref{fig:seedtree} for an example).

To model uncertainty in phylogenetic reconstruction from sequence variation, we simulated sequence evolution along each birth-death tree with INDELible version 1.03 \citep{Indelible09} under an HKY85 \citep{HKY85} model of nucleotide substitution.
Parameters for the substitution model were set to empirical estimates derived from previously published HIV within-host data sets \citep{McCloskey14}. 
Specifically, the stationary distribution was set to 0.42, 0.15, 0.15, 0.28 for the nucleotides A, C, G and T, respectively, and the transition bias parameter was set to 8.5.

Using this approach, we generated a total of 50 replicate subjects without viral latency. 

%A phylogeny was reconstructed from the simulated sequence alignments using the approximate maximum likelihood heuristic implemented in FastTree2.
%Similar results were obtained with trees reconstructed using RAxML \citep{Raxml14}, which is expected to be more accurate, but can consume orders of magnitude more computational time.

To simulate phylogenies with latent viral lineages, we used an approach similar to  \citet{Immonen14}. 
However, we used the simplifying assumptions that any sampled virus lineage could undergo no more than one latent period, and that once a lineage entered a latent state, it remained latent until becoming sampled.
For each simulated `latency-free' phylogeny, we chose 50 tips uniformly at random, then shortened their expected number of substitutions and their branch time (the time between the tip, and the tip's parent).

To do this we applied the following procedure:
\begin{enumerate}
\item Draw a random period for becoming unlatent, $U$, from an exponential distribution with rate, $u$.
\item Draw a random become latent time, $L$, from a restricted exponential distribution with rate, $l$, and bounds $\max\{S-U, 0\}$ and $S$, where $S$ is the branch time of the tip.
\item Shorten the branch time to equal $L$.
\item Scale the expected number of substitutions.
\end{enumerate}
See Figure \ref{fig:latencymodel} for an illustration of this model. We computed $u$ from the mean half life estimated by \cite{Crooks15} ($3.6$ years). Using this estimate we computed $u = 0.00053$.
% by a scaling factor drawn at random from a beta distribution with parameters $\alpha=1$ and $\beta=100$.
We then recorded both the simulated become latent time and simulated collection time and generated sequences along these latent seed trees with INDELible using the same parameters as above. 
For these data, we generated an additional x subjects with 100 sequences each, bringing the total amount of simulated data to x+50 subjects.

\subsection * {Phylogenetic Reconstruction} \label{subsec:phylo}
In our experiments, our reconstructed phylogenies were maximum likelihood phylogenies reconstructed in FastTree2 \citep{FastTree10} using the GTR model.
Similar results were obtained with trees reconstructed using RAxML \citep{Raxml14}, which is expected to be more accurate, but can consume orders of magnitude more computational time.
We will only show the results using RAxML.

We evaluated two different methods for rooting these trees. 
The first method was outgroup rooting, in which one or more sequences from taxa outside but sufficiently related to the group of interest are added to the phylogenetic reconstruction.
The point at which the branch leading to the outgroup intersects the rest of the tree provides an estimate of the root for the latter.
We used the HIV-1 B ancestral sequence reconstruction curated by the LANL database \citep{LosAlamos} as the outgroup sequence.
%% unless we refer to a specific implementation of RTT, there is no reason to claim that our method is "modified"
Second, we used root-to-tip regression \citep[RTT;][]{Korber00}, which performs an exhaustive search by re-rooting the tree at every branch to find the root that minimizes or maximizes an objective function between the sample collection dates and the evolutionary distance of tips from the candidate root.
In our experiments, we used the maximum correlation coefficient as our objective function.
%%RTT regression typically uses dates from all tips.
%For this application, we ignored the tips that were undetermined (either censored or otherwise unknown). 
Our simulated data were only rooted using RTT regression; for all real data, both outgroup rooting and RTT regression were used to root the phylogeny.


\subsection * {Date Reconstruction} \label{subsec:daterecon}
%% Find a way to distinguish this GLM with the one in the next section

Irrespective of the method used to root the phylogeny, we used a general linear model with the normal family to infer the association between the tip dates and sequence divergence from the root. 
The expected amount of evolution was taken to be the response and the sampling dates of the data were taken as the input. 
These models were only calibrated over the uncensored/plasma data for the simulated and real data sets respectively.
%We then used the linear model to predict the dates of the latent/unknown sequences.
To screen for data that were amenable to our analysis, we used the nested log ratio test \citep{Ho14}. 
%To screen for data that were amenable to our analysis, we used Akaike Information Criteria \cite[AIC;][]{Akaike74}.
Our null model was a linear model with zero slope over the expected amount of evolution over time.
%We rejected any sequence data sets whose phylogeny could not reject the null model with a relative AIC of ${-5}$.
We rejected any sequence data sets whose phylogeny couldn't reject the null model with threshold $\alpha=0.01$, since this implied a lack of phylogenetic signal.

We then used the linear model to predict the date of the censored sequences.

\subsection * {Experiments} \label{subsec:experiments}
%We calculated three error measures to quantify the difference between the predicted and known dates of sampling: the mean and median differences, and the root mean square error (RMSE). 
%We used the root mean square error to quantify the difference between the predicted and known dates of sampling.
%the mean square error for predicted date, as well as the mean and median difference between the predicted and known sampling dates.
%and sampling date, where difference defined as simply the predicted date minus the observed date.
%All tip dates were re-scaled to lie between $0$ and $1$ to facilitate comparisons among longitudinal data sets using the formula $ \hat{t} = \frac{t - \min(\bf{t}) }{ \max{(\bf{t})} - \min{(\bf{t}})}$ where $t$ is the the unscaled time of any tip, and $\bf{t}$ is the vector of all tip dates for a given tree. 
%We refer to $\hat{t}$ as the `normalized time' of a tip. 
%Our error metrics were then calculated from these normalized times.%, giving normalized mean square error, normalized mean difference, and normalized median difference. 

%Our first validation experiment on simulated data was performed as follows.
%First, we chose 50\% of the tips of a simulated phylogeny and censored those dates.
%We next used RTT regression \citep{APE} on the remaining data to calibrate the phylogeny's molecular clock. 
%For each censored date, we used the regression model to predict the expected date and then calculated the three error metrics described above.
%For the simulated latent data, we collected the normalized mean square error, normalized mean difference, and normalized median difference. 
%We also collect these for the known `archival' date.
%
%To evaluate the effectiveness of this methodology on real data, we utilized our {\em plasma} data set \citep{McCloskey14} and performed similar experiments. 
%As before, we censored 50\% of their known sampling dates then reconstructed the censored dates using a linear model calibrated over the uncensored dates. 
%We also performed this experiment with RTT regression rooting and out-group rooting. The goal of this experiment was to parallel the type of results of the first experiment on simulated data. 
%
%Finally, we looked at our {\em mixed} data set comprised of both PBMC and plasma data. 
%We used both RTT regression and out-group rooting to root the phylogeny, then calibrated a clock to the known plasma sampling dates and reconstructed the expected age of the sequences for all patients who could reject the null model. 
%Here we only collected the normalized errors between the prediction and the collection.
%

We evaluated the latency of the censored sequences using two methods. The first was a {binomial test} and the second was a {GLME model}.

For the {binomial test}, we used R's binom.test function to apply a right-tailed binomial test on the ocurrence of sequences with positive difference (i.e.~latency). We used $p=0.5$ as the null model.

For the {GLME model}, $s$, we computed the difference, $d_s$, between the sampled date and the predicted date of the linear model. For each data set we used a general linear mixed effects model with the normal family to infer an association between the genetic distance of $s$ from the root of its phylogenetic tree and $d_s$. We specified random intercept and slope for each subject.

We then transformed the difference, $d_s$, by
\[\hat{d}_s = \operatorname{sign}(d_s)\sqrt{|d_s|}\]
and applied the general linear mixed effects model above with $\hat{d}_s$ as the response variable.

\section * {Results} \label{sec:results}

%% Rewrite this whole section

\subsection * {Model Validation} \label{sec:sim_results}
For all of the simulated subjects, we were able to reject the null hypothesis of the date reconstruction regression.

The {binomial test} for the simulated data without viral latency gave a $p$-value of $0.3085$ and a predicted success probability of $0.5052$ with a confidence interval of $(0.4886, 1)$.

%Figure \ref{fig:results1} shows our model validation results over our synthetic data --- here we censor 50\% of the dates, and attempt to reconstruct them. 
%Under the assumption of a strict molecular clock (that evolution is linearly related to time) the clock is a reliable source of information for these simulations.
%Figure \ref{fig:results1} A shows this result for the synthetic data with no latent behaviour.
%Averaged over all simulated data sets, the mean square error was $3.0\times 10^{-3}$, and the average median and mean difference were 0 and $5.0\times 10^{-5}$ respectively. 
%Quantitatively, the censored data points are close to the regression line, and the difference density is heavily peaked around 0.

For the simulated data with viral latency, we computed the difference between the reconstructed date and the sampled date and the difference between the reconstructed date and the become latent date.

%Figure \ref{fig:results1} B shows different results for the latent simulations --- here we censor 50\% of the dates, attempt to reconstruct them, then compare them to the simulated collection dates. 
%In this case, averaged over all simulated data sets, the mean square error was $7.1\times 10^{-3}$, and the average median and mean difference were $4.0\times 10^{-2}$ and $5.1\times 10^{-2}$ respectively. 
%This bias is expected, as the errors were calculated between the simulated collection times --- not the simulated archival times.
%More quantitatively, we expect the density plot to be shifted to the right (as it is in figure \ref{fig:results1} B) as this suggests that sequences are older than their sampling dates.
%When we performed the same analysis against the simulated archival times, the results were nearly identical to the results without any latent data, suggesting that we can recover the dates of latent sequences provided an accurate clock.

For our final validation, we took the same strategy as the first validation on synthetic data, and applied it to our {\em plasma} data set --- we censored 50\% of the known sampling dates, then attempted to reconstruct them.
We were able to reject the null hypothesis in every {\em plasma} subject.

The {binomial test} for the \emph{plasma} data gave a $p$-value of $0.1468$ and a predicted success probability of $0.5392$ with a confidence interval of $(0.4792, 1)$.
%(Figure \ref{fig:results2} C shows this when the phylogenies have been rooted with outgroup rooting). 
%Averaged over all simulated data sets, the mean square error was $1.7\times 10^{-1}$, and the absolute value of average median and mean difference were $2.1\times 10^{-4}$ and $1.9\times 10^{-2}$ respectively. 
%The density plot shows similar behavior to that of the synthetic data, except with a wider distribution, and more variation. 

\subsection * {Application} \label{sec:mixed_data}

%Finally, we tested our methodology on the {\em mixed} data set.
%Figure \ref{fig:results2} D shows the regression over the plasma calibration dates the plasma collection dates. 
%In this case, we do not know the actual date of archival, however, the superposition of the difference density plots shows similar behavior to that of the latent simulated data.

%With respect to the sampling time, over all individuals, the mean square error was $4.9\times 10^{-2}$, and the absolute value of average median and mean difference were $2.4\times 10^{-2}$ and $4.7\times 10^{-3}$.

%Individuals that had received treatment at some point generally failed the hypothesis test --- there was only one patient that did not --- these individuals were not included in the above analysis. 

Finally, we tested our methodology on the {\em mixed} data set. Overall, we found that the RTT regression performed better or just as well as the outgroup rooting. Hence from now on, we only consider the results using RTT regression.

Of the 16 \emph{mixed} subjects, 4 failed to reject the null hypothesis of the date reconstruction regression.

The {binomial test} gave a $p$-value of $0.02263$ and a predicted success probability of $0.5294$ with a confidence interval of $(0.5052, 1)$.

The residuals of the {GLME model} without using the transformation were not normally distributed and thus we will only consider the GLME model using the transformed differences, $\hat{d}_s$.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section * {Discussion} \label{sec:discuss}
%% move to the correct place
Interestingly for the date reconstruction, all of the regressions which produced a negative slope could not reject the null hypothesis. A negative slope for the regession implies that the sequences are becoming less diveregent from the root over time, which is not expected. This suggests that there is a lack of signal in these subjects.

%% Rewrite this section

The results concerning the simulated data were highly consistent with expectations. 
The more a phylogeny adheres to a molecular clock, the more accurate predictions made from it will be --- the simulated data is very well informed, and the phylogenies and their clocks are well resolved by the maximum likelihood tree estimate. 
Since latency is akin to ``pausing'' the evolution of a sequence, and evolution is assumed to be clock-like when sequences are not ``paused'', it is also not surprising that the latent simulated data could be reconstructed with low error. 

The results from the {\em plasma} data-set are also expected. 
Each patient from our plasma data-set showed evidence of a molecular clock, so the ability to reconstruct known dates came as no surprise. 
However, the reconstruction error was higher than that of the simulated data, which is expected for data that come from a real biological process.

Quantitatively, it is difficult to assess the accuracy of the reconstructions within the {\em mixed} data set. 
Many patients show evidence of latency in their regression plots (see the regression plot in Figure \ref{fig:results2} D) but it is unclear what the general pattern is (if any) for the distribution of the error metrics. 
Moreover, it is not possible to know the actual error for this data set, as that would require us to know the date at which a sequence became latent. 
We also observed an interesting sigmoidal pattern in many of the patients in this case.
However, we do note that most treated patients failed the hypothesis testing, implying that samples after treatment should be ignored in any analysis with this methodology.

%Typically, we also observed that the rooting method had very little effect on the errors (the supplemental material has Figure \ref{fig:results2} replicated for RTT rooting). 
%We did, however, identify a case in which RTT rooting fails over outgroup rooting (Figure \ref{fig:degenerate_example} shows an example of this). 
%Specifically, it is possible for the calibration dates to provide insufficient data to resolve the root of the tree. 
%We theorize that this is, for patient 820, because their calibration samples are strongly localized in time, which allows too much freedom (temporally) over the possible roots of the tree. 

%We did not attempt to use BEAST \citep{BEAST} to reconstruct any phylogenies from our data.
%Leaving dates unspecified on the number of tips would have drastically increased the dimensionality of the problem, and we would therefore need a prohibitively large amount of samples from the MCMC chain to assure convergence to the posterior distribution.



We also noticed that the patients who had begun treatment (regardless of whether or not they regressed) typically failed the hypothesis testing. Upon analysing the clocks of these individuals, they showed two trends. 
The first trend, in patients who had regressed after treatment, was the ``resetting'' of the molecular clock --- that is, after treatment started, the clock was set to a previous point in time, then continued as it had before after treatment failed. 
For patients who had successfully been treated, there appeared to be an emergence of a clock with near zero slope after treatment.
This suggests that after a patient begins treatment, the assumption of a molecular clock  is not valid over the whole temporal range of the phylogeny. 



%\section * {Conclusion} \label{sec:conclusion}
In this work we showed that a phylogeny's clock, when reliable, is a valuable source of information with respect to dating sequences that are suspected to show latent behaviour in patients infected with HIV-1. 
Through simulation, we built phylogenies that adhered to clock-like behaviour, with and without latent sequences.
From those, were able to reliably reconstruct dates for both active and latent sequences, rooting the trees via root-to-tip regression.
Then, using real data, we performed analogous reconstructions, using both root-to-tip regression and outgroup rooting, and showed similar results.

However, there are ways in which this method could be improved. 
These results hinge on the strong assumption of a molecular clock over the data, which is not always true. 
It may be possible to adapt a maximum likelihood relaxed clock methodology with this approach, giving more flexibility over noisy or unreliable data. 
Additionally, we would like to apply the methodology to next gen sequence data --- we would like to see if mass amounts of short read data contain sufficient information to calibrate the clock.


%\section{Acknowledgments} \label{sec:ackn}
%The authors would like to thank --.

\clearpage

\bibliographystyle{plainnat}
\bibliography{blind-dating}


\clearpage

%%%%%%%%%%%%%%%  FIGURES  %%%%%%%%%%%%%%%

\section * {Figures}

\begin{figure}[ht]
	\centering
	%\scalebox{5}{\input{figures/example.tex}}
	\includegraphics{figures/latency-scheme}
	\caption[]{ 
	Reconstructing the time that a viral lineage entered a latent state from sequence variation.  
	A dashed line illustrates the linear relationship between the divergence of lineages from the ancestral sequence at the root ($y$-axis) and passage of time since the root ($x$-axis).
	Grey lines represent the reconstructed phylogenetic relationships among these lineages.
	The lineages were sampled (open circles) at three points in time.
	%Above is a time calibrated phylogeny with three taxa, the dotted line in is an example of latency. 
	One of these lineages had become latent at an earlier point in time (red hexagon).
	This lineage subsequently underwent negligible molecular evolution until it was sampled as integrated viral DNA (dashed circle).
	If the relationship between sequence divergence and time is sufficiently linear, then the time that the lineage became latent can be inferred from its sequence, here represented by the thick red dashed line.
	%Sequence A was archived at time $t$, and was collected at the same time as sequence B, at time $t^\prime$ (a drift of $t^\prime - t$ is expected).
	%If the molecular clock assumption holds true, the MRCA is known, and B and C are reliable time-points, then the date at $t$ can be inferred from from the expected amount of evolution for B from the root.
	}
	\label{fig:latenttree}
\end{figure}

\begin{figure}[ht]
	\centering{}
	\includegraphics[width=10cm]{figures/latency-model}
	\caption[Latency Model]{An illustration of the latency model. On a tip with branch length $S$, we randomly select a become unlatent period, $U$, and then random select a become latent period, $L$, between the interval $S-U$ and $S$. In the illustration, the red dot represents when the lineage begins to go latent, the green dot represents when the lineage is sampled, and the blue dot represents when the lineage would have been reactivated if it was not sampled. }
	\label{fig:latencymodel}
\end{figure}

\begin{figure}[p]
	\centering
	\includegraphics[width=\textwidth]{figures/simulated.pdf} 
	\includegraphics[width=\textwidth]{figures/simulated_latent.pdf}\\
	\caption[Simulated Data]{On the left of each case is a plot of evolutionary distance vs time, the right is the super position of density estimates (of normalized error) for each phylogeny (with kernel bandwidth = $0.15$). The red line on the density plot is the mean error, and the black line is the median error. }
	 \label{fig:results1}
\end{figure}

\begin{figure}[ht]
	\centering
	\includegraphics[width=\textwidth]{figures/ancre.pdf}
	\includegraphics[width=\textwidth]{figures/lanl.pdf}
	\caption[Examples]{As Figure \ref{fig:results1}, except for the real data.
The phylogenies in these cases have been rooted with outgroup rooting}
	\label{fig:results2}
\end{figure}

\begin{figure}[ht]
	\centering
	\includegraphics[width=\textwidth]{figures/GLM_plot.pdf}
	\caption[Regression]{A plot of the genetic distance from the root versus the difference from the estimate of the date reconstruction. The coloured circles respresent the values for each DNA sequence for each subject (each subject has their own colour). The solid black line is the quadratric regression with the mean of the parameters from the mixed effects model.  The dotted lines are the quadratric regression with one standard deviation in both parameters.}
	\label{fig:glm}
\end{figure}

%\begin{figure*}[ht]
%	\centering
%	\includegraphics[scale=0.425]{figures/rtt.pdf} \\
%	\caption[Example of bad root]{Root to tip rooting fails due to insufficient temporal resolution at the root}
%	\label{fig:degenerate_example}
%\end{figure*}


\clearpage

%%%%%%%%%%%%%%%  TABLES  %%%%%%%%%%%%%%%

\section * {Tables}

\begin{table*}[!ht]
\def\arraystretch{1.3}%
\begin{center}
\begin{tabular}{llrrrrrrr} 

Reference & Patient ID & \multicolumn{3}{c}{Sequences} & \multicolumn{3}{c}{Time points} & Reject Null \\ 
 &  & Plasma & PBMC & Total & Plasma & PBMC & Total\\
\hline
\cite{Shankarappa99}% & 820 &       50 &       87 &      137 &        5 &       10 &       15  \\
& 821 &      76 &      192 &      268 &        7 &       17 &       17 &   Yes \\ 
& 822 &      32 &       98 &      130 &        3 &       10 &       10 &   Yes \\ 
& 824 &     100 &      107 &      207 &        7 &        9 &       13 &   Yes \\ 
& 10137 &     24 &      106 &      130 &        2 &       10 &       12 &   No \\ 
& 10138 &     82 &      119 &      201 &        6 &       13 &       16 &   Yes \\
%& 10586 &     16 &      121 &      137 &        2 &       12 &       14  \\ 
& 13889 &    151 &      132 &      283 &       13 &       14 &       18 &   Yes \\ 
% \cite{Fischer04} & 10769 &    229 &       96 &      325 &       10 &        4 &       11 &   Yes \\ 
%& 10770 &    190 &       31 &      221 &       11 &        2 &       11  \\ 
\cite{Llewellyn06} & 16616 &     16 &       95 &      111 &        2 &        5 &        5 &    Yes  \\ 
& 16617 &     16 &       89 &      105 &        2 &        5 &        5 &   Yes \\ 
& 16618 &     25 &       75 &      100 &        2 &        5 &        6 &   Yes \\ 
& 16619 &     13 &       60 &       73 &        2 &        6 &        6 &   Yes \\ 
\cite{Novitsky09}% & 34382 &     37 &        5 &       42 &        3 &        1 &        4  \\ 
& 34391 &     13 &       38 &       51 &        3 &        5 &        6 &   No \\ 
& 34393 &     25 &       74 &       99 &        2 &        6 &        7 &   Yes  \\ 
& 34396 &      23 &       46 &       69 &        2 &        5 &        5 &   No \\ 
%& 34397 &      26 &       25 &       51 &        2 &        2 &        4  \\ 
& 34399 &      61 &       88 &      149 &        5 &        9 &       12 &   Yes \\ 
%& 34400 &      54 &       23 &       77 &        4 &        1 &        5  \\ 
%& 34405 &      27 &       29 &       56 &        3 &        2 &        4  \\ 
& 34408 &       5 &       73 &       78 &        2 &        8 &        8 &    No \\ 
& 34410 &      35 &       60 &       95 &        2 &        6 &        6 &    Yes \\ 
%& 34411 &      25 &       43 &       68 &        3 &        3 &        6   \\ \hline
\end{tabular}
\end{center}
  \caption{Summary of all the patient data collected from the HIV LANL database --- Patient ID corresponds to the Los Alamos database's Patient ID \citep{LosAlamos}.
   }\label{tab:patients} 
\end{table*}



\end{document}


