% !TEX TS-program = xelatex
\documentclass[12pt]{article}

\usepackage[margin=2cm]{geometry}
\usepackage{colortbl}
\usepackage{comment}
\usepackage{caption}
\usepackage{subcaption}
\usepackage{mathptmx}
\usepackage{nicefrac}
\usepackage{authblk}
\usepackage{array}


\usepackage{times}
%\usepackage{lineno}
%\usepackage[round]{natbib}
%\makeatletter
%\renewcommand{\@biblabel}[1]{#1.}
%\makeatother

\usepackage{graphicx}
\usepackage{amsmath}
\usepackage{xcolor}

\usepackage{url}
\urlstyle{same}

\usepackage[small,compact]{titlesec}

\renewcommand{\thefigure}{S\arabic{figure}}
\renewcommand{\thetable}{S\arabic{table}}
\renewcommand{\thesection}{S\arabic{section}}

%\titleformat{\section} {\vspace{24pt}\bf\sffamily\MakeUppercase}{\thesection} {0pt} {}
\titleformat{\section} {\vspace{12pt}\bf\large}{\thesection\ }{0pt}{}
\titleformat{\subsection} {\vspace{6pt}\bf}{\thesubsection} {0pt} {\vspace{-2pt}}
\titleformat{\subsubsection} [runin] {\bf}{\thesubsubsection} {12pt} {}

\newcolumntype{+}{>{\global\let\currentrowstyle\relax}}
\newcolumntype{^}{>{\currentrowstyle}}
\definecolor{badpatcol}{HTML}{FF6666}
\newcommand{\badpat}{\gdef\currentrowstyle{\bfseries}\bfseries\ignorespaces }

\captionsetup[subfigure]{width=0.9\textwidth}

\newcommand{\high}[1]{{\textcolor{badpatcol} #1}} 
 
\begin{document}

\title{Supplementary Materials \\ Blind-dating: recovering the integration dates of the latent human immunodeficiency virus reservoir using longitudinal samples}

% Use letters for affiliations, numbers to show equal authorship (if applicable) and to indicate the corresponding author
\author[a]{Bradley R. Jones}
\author[b]{Natalie N. Kinoch} 
\author[a]{Joshua Horacsek}
\author[b]{Philip Mwimanzi}
\author[b]{Bemuluyigza Baraki}
\author[c]{John Huang}
\author[c]{Ronald Truong}
\author[a]{Bruce Ganase}
\author[a]{Marianne Harris}
\author[a,d]{P. Richard Harrigan}
\author[c]{R. Brad Jones}
\author[a,b]{Mark A. Brockman}
\author[a,d]{Jeffrey B. Joy}
\author[e]{Art F. Y. Poon}
\author[b]{Zabrina L. Brumme}

\affil[a]{BC Centre for Excellence in HIV/AIDS, 608-1081 Burrard St Vancouver, Canada V6Z 1Y6}
\affil[b]{Faculty of Health Sciences, Simon Fraser University, 8888 University Drive
Burnaby, Canada V5A 1S6}
\affil[c]{Department of Microbiology, Immunology and Tropical Medicine, George Washington University, Ross Hall 2300 Eye Street, NW, Suite 502 Washington, DC, United States of America 20037}
\affil[d]{Department of Medicine, University of British Columbia, 2775 Laurel Street, 10th Floor
Vancouver, Canada V5Z 1M9}
\affil[e]{Department of Pathology and Laboratory Medicine, Western University, Dental Sciences Building, Rm. 4044
London, Canada N6A 5C1}
%%% 1 Simon Fraser University
%%% 2 BC Centre for Excellence in HIV/AIDS
%%% 3 University of British Columbia
\baselineskip 22pt
%\pagewiselinenumbers

\date{}
\maketitle

\section{Simulations}

Small intro

\subsection*{Methods}
\subsection*{Simulated data}
To characterize sources of error in reconstructing dates on tips of a phylogeny, we generated phylogenetic trees by direct simulation. % and from the plasma data sets.
First, we generated phylogenies with 100 tips each under a birth-death model with serial sampling using the \emph{sim.bdsky.stt} function in the \textit{R} package \textit{TreeSim} \cite{Boskova14}.
Birth-death models have been applied to studying speciation processes \cite{Nee06} and infectious disease epidemics \cite{Stradler13}, but have only recently been used to model the proliferation of virus lineages in cell populations \cite{Hartfield15}.
%% This paragraph needs to be rewritten
The birth-death model was parametrized by fitting this model to samples from the {\em plasma} dataset.
%% we may need to provide some justification for using BD
%To assess the sensitivity of our method, we first simulated HIV sequence data sets without viral latency.
We used BEAST version 2.3.2 \cite{BEAST2} to estimate the posterior distribution defined by the serial model with a Bayesian skyline \cite{Stadler13}, a strict molecular clock, and the HKY85 \cite{HKY85} model of nucleotide substitution.
%Using this approach, we generated samples of trees for each sequence alignment in the plasma dataset.
%From these simulations, we evaluated the accuracy of predicting sample collection times from a strict molecular clock model that was fit to a training subset of these data.
The lineage birth rate, death rate and sampling probability were estimated to be: $\lambda = 5.12 \times 10^{-2}$ day$^{-1}$, $\delta = 5.01 \times 10^{-2}$ day$^{-1}$, $s = 5.24 \times 10^{-3}$ respectively.
We used the \textit{R} package, \emph{NELSI} \cite{NELSI}, to assign rates of evolution to these trees by drawing values from a Gaussian distribution with a mean rate $\mu = \ 1.96\times 10^{-4}$ substitutions per generation andstandard deviation $\sigma = \ 1.42\times 10^{-5}$. %; Figure \ref{fig:seedtree} for an example).
To model uncertainty in phylogenetic reconstruction from sequence variation, we simulated sequence evolution along each birth-death tree with INDELible version 1.03 \cite{Indelible} under an HKY85 \cite{HKY85} model of nucleotide substitution.
Parameters for the substitution model were set to empirical estimates derived from previously published HIV within-host data sets \cite{McCloskey14}. 
Specifically, the stationary distribution was set to 0.42, 0.15, 0.15, 0.28 for the nucleotides A, C, G and T, respectively, and the transition bias parameter was set to 8.5.
Using this approach, we generated a total of 50 replicate subjects.

\subsection*{Results}

\section{Outgroup rooting}
In addition to root-to-tip regression (RTT) we also evaluated using out group rooting (OGR), in which one or more sequences from taxa outside but sufficiently related to the group of interest are added to the phylogenetic reconstruction \cite{Huelsenbeck02}, in order to root the maximum likelihood (ML) phylogenies.
The point at which the branch leading to the outgroup intersects the rest of the ML phylogeny provides an estimate of the root for the latter.
We selected the HIV-1 B ancestral sequence reconstruction, HXB2 (accession: K03455), curated by the LANL HIV sequence database, as the outgroup sequence.

\subsection*{Results with simulations and public data sources}

\subsection*{Results recruited patients}

\subsection*{Comparison to rooting with root-to-tip regression}
OGR performed well, but inferiorly compared to RTT.
Results in all data sets showed higher RMSEs for the training sets and more variable RMSEs and concordance in the validation studies.
Nonetheless the concordance between RTT and OGR was very high.
There was a 0.934 concordance \cite{Lin89} between the estimated dates of the DNA sequences in the LANL patients using RTT versus using OGR (see Supplementary Figure \ref{fig:lanlconcord}).
The concordance in the recruited patients was also high with a 0.899 concordance in patient 1 and a 0.911 concordance in patient 2 (see Supplementary Figure \ref{fig:concord}).

\begin{table*}
\caption{Summary of all the patient data collected from the LANL HIV sequence database \cite{LosAlamos} in the data sets from public sources.\label{tab:patients}}
\def\arraystretch{1.3}%
\begin{center}
\begin{tabular}{llrrrrrrr} 

Reference & Patient ID & \multicolumn{3}{c}{Sequences} & \multicolumn{3}{c}{Time points} \\
 &  & Plasma & PBMC & Total & Plasma & PBMC & Total\\
\hline
\cite{Simmonds91} & 2658 & 69 (84) & & 69 (84) & 5 & & 5 \\
\cite{Shankarappa99} & 825 & 49 (52) & & 49 (52) & 6 & & 6 \\
\cite{Edwards06} & 7259 & 28 & & 28 & 4 & & 4 \\
%& 7263 & 12 (14) & & 12 (14) & 3 & & 3 \\	% bad before censoring
& 7265 & 40 (21) & & 40 (21) & 3 & & 3 \\
& 13333 & 38 (39) & & 38 (39) & 4 & & 4 \\
& 13334 & 36 (38) & & 36 (38) & 5 & & 5 \\
& 13336 & 42 & & 42 & 4 & & 4 \\
\cite{Schnell10} & 35566 & 81 (92) & & 81 (92) & 2 & & 2 \\
\hline
\cite{Shankarappa99} & 820 & 45 (50) & 81 (87) & 126 (137) & 5 & 10 & 15 \\
& 821 & 69 (76) & 178 (192) & 247 (268) & 7 & 17 & 17 \\ 
& 822 & 29 (32) & 90 (98) & 119 (130) & 3 & 10 & 10 \\ 
& 824 & 52 (100) & 102 (107) & 154 (207) & 7 & 9 & 13 \\
& 10137 & 17 (24) & 71 (106) & 88 (130) & 2 & 10 & 12 \\
& 10138 & 64 (82) & 112 (119) & 176 (201) & 6 & 13 & 16 \\
& 10586 & 11 (16) & 65 (121) & 76 (137) & 2 & 12 & 14 \\ 
& 13889 & 77 (151) & 65 (132) & 142 (283) & 13 & 14 & 18 \\
\cite{Fischer04} & 10769 & 108 (229) & 56 (96) & 164 (325) & 10 & 4 & 11 \\ 
%& 10770 &    190 &       31 &      221 &       11 &        2 &       11  \\ 
\cite{Llewellyn06} & 16616 & 16 & 64 (95) & 80 (111) & 2 & 5 & 5 \\
%& 16617 & 7 (16) & 62 (89) & 69 (105) & 1 (2) & 5 & 5 \\
& 16618 & 18 (25) & 45 (75) & 63 (100) & 2 & 5 & 6 \\
& 16619 & 9 (13) & 45 (60) & 54 (73) & 2 & 6 & 6 \\
\cite{Novitsky09}% & 34382 &     37 &        5 &       42 &        3 &        1 &        4  \\ 
& 34391 & 12 (13) & 35 (38) & 52 (51) & 3 & 5 & 6 \\
& 34393 & 21 (25) & 31 (74) & 47 (99) & 2 & 5 (6) & 7 (6) \\
& 34396 & 21 (23) & 20 (46) & 52 (69) & 2 & 5 & 5 \\
%& 34397 &      26 &       25 &       51 &        2 &        2 &        4  \\ 
& 34399 & 50 (60) & 72 (88) & 122 (148) & 5 & 9 & 12 \\
%& 34400 &      54 &       23 &       77 &        4 &        1 &        5  \\ 
%& 34405 &      27 &       29 &       56 &        3 &        2 &        4  \\ 
%& 34408 &       5 &       73 &       78 &        2 &        8 &        8 \\
& 34410 & 22 (35) & 56 (60) & 78 (95) & 2 & 6 & 6 \\
& 34411 & 14 (25) & 19 (42) & 33 (67) & 3 & 3 & 6 \\
\hline
\end{tabular}
\end{center}
	Patient ID corresponds to the anonymized patient identifiers in the LANL database.
	Values in parentheses indicate the quantity before data cleaning.
\end{table*}

\pagebreak{}

\begin{table*}
\caption{Details of the linear models applied to the LANL data sets.\label{tab:patientserror} }
%\tabletitle{Linear model details}
\def\arraystretch{1.3}%
\begin{center}
\begin{tabular}{+l^r^r^r^r} 
Patient ID & Model AIC & Null AIC & MA(E/D) & SMA(E/D) \\ 
\hline
2658 & -185 & -121 & 234 & 0.124 \\
825 & -150 & -89.6 & 325 & 0.108 \\
\badpat 7259 & -97.6 & -87.8 & 224 & 0.253 \\
%\badpat 7263 & -42.7 & -41.8 & 136 & 0.541 \\ % bad before censoring
\badpat 7265 & -50.7 & -42.2 & 122 & 0.444 \\
13333 & -133 & -115 & 111 & 0.219 \\
13334 & -127 & -116 & 160 & 0.218 \\
13336 & -91.6 & -63.1 & 123 & 0.166 \\
35566 & -351 & -338 & 155 & 0.596 \\
\hline
820 & -289 & -270 & 2,980 & 0.960 \\
821 & -303 & -122 & 286 & 0.120 \\
822 & -190 & -120 & 325 & 0.153 \\
824 & -279 & -155 & 476 & 0.152 \\
\badpat 10137 & -39.0 & -37.2 & 1,150 & 0.411 \\
10138 & -199 & -119 & 444 & 0.133 \\
10586 & -67.8 & -52.1 & 3,700 & 0.784 \\
13889 & -305 & -175 & 640 & 0.132 \\
\badpat 10769 & -435 & -429 & 2,770 & 1.40 \\
16616 & -111 & -55.1 & 677 & 0.259 \\
%16617 & -91.7 & -45.7 \\
16618 & -110 & -83.5 & 642 & 0.282 \\
\badpat 16619 & -94.5 & -88.5 & 796 & 0.393 \\
34391 & -108 & -87.8 & 88.9 & 0.244 \\
34393 & -202 & -179 & 76.6 & 0.212 \\ 
34396 & -199 & -182 & 67.6 & 0.198 \\
34399 & -289 & -210 & 231 & 0.157 \\
34410 & -192 & -178 & 142 & 0.321 \\
34411 & -131 & -109 & 90.7 & 0.196 \\
\hline
\end{tabular}
\end{center}
	Patient ID corresponds to the anonymized patient identifiers in the LANL database.
	Model AIC is the Akaike Information Criterion (AIC) \cite{Akaike74} of the linear model and Null AIC is the AIC of the null model.
	Patients for whom we were unable to reject the null hypothesis based on a AIC difference less than 10 are marked in bold.
	MAE is mean absolute error, MAD is mean absolute difference (between collection date and estimated date ), SMAE is scaled mean absolute error and SMAD is scaled mean absolute difference.
\end{table*}

\pagebreak{}

\bibliographystyle{pnas-new}
\bibliography{blind-dating}

\end{document}

