\section{Methods} \label{sec:methods}
\subsection{Simulated Data} \label{subsec:simdata}
We first used seed phylogenies generated via a birth death model (with birth and death rate equal to $5\times10^{-3}$ and with a 5\% chance of being sampled) in the R package TreeSim \citep{TreeSim, Stradler13, Boskova14}, and assign a clock to these trees (with rate$=3x10^{-3}$ substitutions per site/day, and with noise pulled from a normal distribution with $\sigma=5\times10^{-5}$.
We then used INDELible 1.03 \citep{Indelible09} to simulated nucleotide sequences along these seed phylogenies with an HKY85 \citep{HKY85} substitution model, and  parameters set from a maximum likelihood estimate of published within host data \citep{McCloskey14}. 
From these simulated sequences a maximum likelihood phylogeny was then reconstructed with RAxML \citep{Raxml14}. In total, we generated 50 trees without latency, each having 100 tips. 

To generate the trees that had observable latency, we took a similar approach to \cite{Immonen14}. 
We randomly selected a branch within a seed phylogeny then scaled it by a random value between $0.1$ and $0.9$, to assure that the phylogeny had sufficiently many latent tips, we rejected all such trees that had less than 40 and more than 60 latent tips. 
We then recorded both the simulated age and simulated collection time, then generated sequences along these latent seed trees with INDELible 1.03 \citep{Indelible09} and the same parameters above. 
From these simulated sequences a maximum likelihood phylogeny was then reconstructed with RAxML \citep{Raxml14}.
For these data, we generated an additional 50 trees with 100 tips, bringing the total amount of simulated data to 100 trees with 100 tips each. 


\subsection{Data Collection} \label{subsec:dcollection}
We use a previously collected and cleaned set of data \citep{McCloskey14} from the Los Alamos HIV Database (\anote{reference}). 
This data set contains 335 data sets from 232 patients, with a total of over 19,000 sequences \citep{McCloskey14} containing mostly samples collected from plasma. 
From this data set we use \anote{X} plasma samples from \anote{Y} specific patients. 
We also directly utilized the Los Alamos Advanced search interface \citep{LosAlamos} to identify subjects with both plasma and PBMC sequences.

The first set of data we utilized were RNA sequences from untreated patients from longitudinal studies with two or more clonal or single-genome (SGS) sequences available at each time point, with a known time-line relative to one of several reference points: HIV infection, seroconversion or presentation of symptomatic seroconversion illness. The samples were collected within 6 months (186 days) of this reference point, and at least one of the subsequent (“follow-up”) time points occurred a minimum of 6 months after baseline. Known cases of superinfection were also excluded, all patients were untreated, and the samples were taken from the env region of the HIV-1 genome \citep{McCloskey14}. We call these our plasma sequences. 
 
For our second data set -- containing both plasma and PBMC samples -- we selected  individuals from published studies that had well-characterized partial env sequences longitudinally sampled from HIV RNA in blood plasma, or sampled from integrated viral DNA in host PBMC cells. There was no condition for the time between sample collections. \anote{X} individuals were included with 
plasma samples from at least 2 time points (table \ref{tab:patients}). A total of \anote{X} partial env sequences from blood plasma, and \anote{Y} partial env sequences from PBMC cells were used in our analysis. We utilized samples from individuals that had not received treatment \citep{Shankarappa99, Novitsky09}, from individuals that had been treated via HAART \citep{Llewellyn06}, and individuals that had been on HAART, but stopped taking it at some point \citep{Fischer04}. 
 
\begin{table*}[!ht]\label{tab:patients} 
\def\arraystretch{1.3}%
\begin{center}
\begin{tabular}{llrrrrrr} 

Reference & Patient ID & \multicolumn{3}{c}{Sequences} & \multicolumn{3}{c}{Time points}\\ 
 &  & Plasma & PBMC & Total & Plasma & PBMC & Total\\
\hline
\cite{Shankarappa99} & 820 &       50 &       87 &      137 &        5 &       10 &       15  \\
& 821 &      76 &      192 &      268 &        7 &       17 &       17   \\ 
& 822 &      32 &       98 &      130 &        3 &       10 &       10   \\ 
& 824 &     100 &      107 &      207 &        7 &        9 &       13   \\ 
& 10137 &     24 &      106 &      130 &        2 &       10 &       12  \\ 
& 10138 &     82 &      119 &      201 &        6 &       13 &       16  \\
& 10586 &     16 &      121 &      137 &        2 &       12 &       14  \\ 
& 13889 &    151 &      132 &      283 &       13 &       14 &       18  \\ 
 \cite{Fischer04} & 10769 &    229 &       96 &      325 &       10 &        4 &       11  \\ 
& 10770 &    190 &       31 &      221 &       11 &        2 &       11  \\ 
\cite{Llewellyn06} & 16616 &     16 &       95 &      111 &        2 &        5 &        5  \\ 
& 16617 &     16 &       89 &      105 &        2 &        5 &        5  \\ 
& 16618 &     25 &       75 &      100 &        2 &        5 &        6  \\ 
& 16619 &     13 &       60 &       73 &        2 &        6 &        6  \\ 
\cite{Novitsky09} & 34382 &     37 &        5 &       42 &        3 &        1 &        4  \\ 
& 34391 &     13 &       38 &       51 &        3 &        5 &        6  \\ 
& 34393 &     25 &       74 &       99 &        2 &        6 &        7  \\ 
& 34396 &      23 &       46 &       69 &        2 &        5 &        5 \\ 
& 34397 &      26 &       25 &       51 &        2 &        2 &        4  \\ 
& 34399 &      61 &       88 &      149 &        5 &        9 &       12  \\ 
& 34400 &      54 &       23 &       77 &        4 &        1 &        5  \\ 
& 34405 &      27 &       29 &       56 &        3 &        2 &        4  \\ 
& 34408 &       5 &       73 &       78 &        2 &        8 &        8 \\ 
& 34410 &      35 &       60 &       95 &        2 &        6 &        6 \\ 
& 34411 &      25 &       43 &       68 &        3 &        3 &        6   \\ \hline
\end{tabular}
\end{center}
  \caption{{Summary of all the patient data collected from the HIV LANL database -- Patient ID corresponds to the Los Alamos database's Patient ID \citep{LosAlamos}.}
   }
\end{table*}

\subsection{Sequence Aligment} \label{subsec:seqalign}
All of our sequences were annotated in FASTA format, each containing multiple time points and both plasma and PBMC sequences (if applicable). Our simulated sequences were not simulated with any insertions or deletions, and no alignment was necessary. Additionally, our plasma data set \citep{McCloskey14} had already been aligned and cleaned. For the plasma and PBMC dataset, we used the built in MUSCLE \citep{Muscle04} interface within AliView \citep{AliView14} to align the sequences, which were then visually inspected and cleaned. 

\subsection{Phylogenetic Reconstruction} \label{subsec:phylo}
In all experiments, our reconstructed phylogenies were maximum likelihood phylogenies reconstructed in RAxML \citep{Raxml14} using the GTR+$\Gamma$ model. Synthetic data were solely rooted with root-to-tip regression \citep{APE}. For all other patient data, both outgroup rooting (against the HIV-1 B ancestor consensus  \anote{ref}) and root-to-tip regression were used to root the tree. We did not attempt to use BEAST \citep{BEAST} to reconstruct any phylogenies from our samples. Leaving dates unspecified on the number of tips we had would drastically increase the dimensionality of the problem, and we would therefore need a prohibitively large amount of samples from the MCMC chain to assure we found the desired tree.

To root these trees 
For tree rooting, we use a modified version of root-to-tip regression that only considers a subset of calibration data \citep{APE}. 
\subsection{Date Reconstruction} \label{subsec:daterecon}
Once the trees had been rooted, to reconstruct dates, a general linear model with the normal family was constructed. The expected amount of evolution was taken to be the response and the sampling dates of the data were taken as the input. These models were only trained over the uncensored/plasma data for the simulated and real data sets respectively.

\subsection{Patient Rejection} \label{subsec:hypot}
To screen for patients whose data are amenable to our analysis, we used the nested log ratio test \citep{Ho14}. Our null model was a linear model with zero slope over the expected amount of evolution over time. We rejected any patients whose phylogeny couldn't reject the null model with threshold $\alpha=0.01$.

