\section{Methods} \label{sec:methods}
%To validate our approach, we first simulated sequence data along a phylogeny with a strict clock. We reconstructed phylogenies and randomly chose tips to censor, then reconstructed those dates from the clock. We then tested our method on patient data that was RNA only, and did the same thing, finally, we look at a homogenous dataset with both plasma and PBMC data. 

\subsection{Simulated Data} \label{subsec:simdata}
We used INDELible 1.03 \citep{Indelible09} to generate sequences along seed phylogenies with an HKY85 \citep{HKY85} substitution model with parameters set from a maximum likelihood estimate. The seed phylogenies were constructed via TreeSim \citep{TreeSim, Stradler13, Boskova14} through a birth death model (birth and death rate equal to $5e-3$ and with a 5\% chance of being sampled.) These trees then have a strict clock assigned to them (with rate=$3e-3$ and noise=$1e-5$). A maximum likelihood phylogeny was then reconstructed from these sequences with RAxML \citep{Raxml14}. When we root these trees, we use a modified version of root-to-tip regression that only considers a subset of calibration data \citep{APE}. In total, we generated 50 trees with 100 tips each. 

To generate the trees that simulated latency, we took a similar approach to \cite{Immonen14}. To simulate the latent behaviour -- collection dates drifting from their expected dates -- we randomly selected up to 5 branches within a phylogeny, and scaled them by a random value, chosen uniformly between $0.1$ and $0.9$. For these data, we generated an additional 50 trees with 100 tips, bringing the total amount of simulated data to 100 trees with 100 tips each. 

\subsection{Data Collection} \label{subsec:dcollection}
We use a previously collected and cleaned set of data \citep{McCloskey14} from the Los Alamos HIV Database (\anote{reference}). This data set contains 335 data sets from 232 patients, with a total of over 19,000 sequences \citep{McCloskey14} containing mostly samples collected from plasma. From this data set we use \anote{X} plasma samples from \anote{Y} specific patients. We also directly utilized the Los Alamos Advanced search interface \citep{LosAlamos} to identify subjects with both plasma and PBMC sequences.

\subsection{Patient Data} \label{subsec:patdata}
The first set of data we utilized were RNA sequences from untreated patients from longitudinal studies with two or more clonal or single-genome (SGS) sequences available at each time point, with a known time-line relative to one of several reference points: HIV infection, seroconversion or presentation of symptomatic seroconversion illness. The samples were collected within 6 months (186 days) of this reference point, and at least one of the subsequent (“follow-up”) time points occurred a minimum of 6 months after baseline. Known cases of superinfection were also excluded, all patients were untreated, and the samples were taken from the env region of the HIV-1 genome \citep{McCloskey14}. We call these our plasma sequences. 
 
For our second data set -- containing both plasma and PBMC samples -- we selected  individuals from published studies that had well-characterized partial env sequences longitudinally sampled from HIV RNA in blood plasma, or sampled from integrated viral DNA in host PBMC cells. There was no condition for the time between sample collections. \anote{X} individuals were included with 
plasma samples from at least 2 time points (table \ref{tab:patients}). A total of \anote{X} partial env sequences from blood plasma, and \anote{Y} partial env sequences from PBMC cells were used in our analysis. We utilized samples from individuals that had not received treatment \citep{Shankarappa99, Novitsky09}, from individuals that had been treated via HAART \citep{Llewellyn06}, and individuals that had been on HAART, but stopped taking it at some point \citep{Fischer04}. 
 
\begin{table*}[!ht]\label{tab:patients} 
\def\arraystretch{1.3}%
\begin{tabularx}{\textwidth}{ X | X | X | X | X | X | X } 
\hline
\hline
Patient ID & \# of plasma samples & \# of PBMC samples & Total \# of Seqeunces & \# Plasma Time points & \# PBMC time points & Total Time points   \\ \hline \hline
820$^1$ &       50 &       87 &      137 &        5 &       10 &       15  \\
821$^1$ &       76 &      192 &      268 &        7 &       17 &       17   \\ 
822$^1$ &       32 &       98 &      130 &        3 &       10 &       10   \\ 
824$^1$ &      100 &      107 &      207 &        7 &        9 &       13   \\ 
10137$^1$ &       24 &      106 &      130 &        2 &       10 &       12  \\ 
10138$^1$ &       82 &      119 &      201 &        6 &       13 &       16  \\
10586$^1$ &       16 &      121 &      137 &        2 &       12 &       14  \\ 
10769$^2$ &      229 &       96 &      325 &       10 &        4 &       11  \\ 
10770$^2$ &      190 &       31 &      221 &       11 &        2 &       11  \\ 
13889$^1$ &      151 &      132 &      283 &       13 &       14 &       18  \\ 
16616$^3$ &       16 &       95 &      111 &        2 &        5 &        5  \\ 
16617$^3$ &       16 &       89 &      105 &        2 &        5 &        5  \\ 
16618$^3$ &       25 &       75 &      100 &        2 &        5 &        6  \\ 
16619$^3$ &       13 &       60 &       73 &        2 &        6 &        6  \\ 
34382$^4$ &       37 &        5 &       42 &        3 &        1 &        4  \\ 
34391$^4$ &       13 &       38 &       51 &        3 &        5 &        6  \\ 
34393$^4$ &       25 &       74 &       99 &        2 &        6 &        7  \\ 
34396$^4$ &       23 &       46 &       69 &        2 &        5 &        5 \\ 
34397$^4$ &       26 &       25 &       51 &        2 &        2 &        4  \\ 
34399$^4$ &       61 &       88 &      149 &        5 &        9 &       12  \\ 
34400$^4$ &       54 &       23 &       77 &        4 &        1 &        5  \\ 
34405$^4$ &       27 &       29 &       56 &        3 &        2 &        4  \\ 
34408$^4$ &        5 &       73 &       78 &        2 &        8 &        8 \\ 
34410$^4$ &       35 &       60 &       95 &        2 &        6 &        6 \\ 
34411$^4$ &       25 &       43 &       68 &        3 &        3 &        6   \\ \hline \hline
\end{tabularx}

  \caption{{Summary of all the patient data collected from the HIV LANL database -- Patient ID corresponds to the Los Alamos database's Patient ID \citep{LosAlamos}.\hfill}\hspace{\textwidth}$^1$ \cite{Shankarappa99}, $^2$ \cite{Fischer04}, $^3$ \cite{Llewellyn06},  $^4$ \cite{Novitsky09}
   }
\end{table*}

\subsection{Sequence Aligment} \label{subsec:seqalign}
All of our sequences were annotated in FASTA format, each containing multiple time points and both plasma and PBMC sequences (if applicable). Our simulated sequences were not simulated with any insertions or deletions, and no alignment was necessary. Additionally, our plasma data set \citep{McCloskey14} had already been aligned and cleaned. For the plasma and PBMC dataset, we used the built in MUSCLE \citep{Muscle04} interface within AliView \citep{AliView14} to align the sequences, which were then visually inspected and cleaned. 

\subsection{Phylogenetic Reconstruction} \label{subsec:phylo}
In all experiments, our reconstructed phylogenies were maximum likelihood phylogenies reconstructed in RAxML \citep{Raxml14} using the GTR+$\Gamma$ model. Synthetic data were solely rooted with root-to-tip regression \citep{APE}. For all other patient data, both outgroup rooting (against the HIV-1 B ancestor consensus  \anote{ref}) and root-to-tip regression were used to root the tree. We did not attempt to use BEAST \citep{BEAST} to reconstruct any phylogenies from our samples. Leaving dates unspecified on the number of tips we had would drastically increase the dimensionality of the problem, and we would therefore need a prohibitively large amount of samples from the MCMC chain to assure we found the desired tree.

\subsection{Date Reconstruction} \label{subsec:daterecon}
Once the trees had been rooted, a general linear model with the normal family was constructed with the with expected number of substitutions as the input, and sampling time as the response. For all experiments using patient data, this fit was only over the plasma data, and the PBMCs expected subs were used as input. From this, we collected the difference between the observed sampling date, and the predicted date. 

\subsection{Patient Rejection} \label{subsec:hypot}
Our data are messy because they come from some biological process. We used the nested log ratio test on to reject patients. Our null model was a linear model with zero slope over the evolution of sequences over time. We rejected any patients who didn’t meet the alpha threshold of 0.01.
