\section{Methods} \label{sec:methods}
To validate our approach, we first simulated sequence data along a phylogeny with a strict clock. We reconstructed phylogenies and randomly chose tips to censor, then reconstructed those dates from the clock. We then tested our method on patient data that was RNA only, and did the same thing, finally, we look at a homogenous dataset with both plasma and PBMC data. 

\subsection{Simulated Data} \label{subsec:simdata}
Sequences were generated in INDELible 1.03 \citep{Indelible09} along a seed phylogeny. This phylogeny was generated in TreeSim \citep{TreeSim}, and was given a clock from NESLI \citep{NELSI}. Sequences were generated using an HKY85 substitution model with parameters set from a maximum likelihood estimate from published datasets \citep{McCloskey14}. A maximum likelihood phylogeny was then reconstructed from these sequences with RAxML \citep{Raxml14}, which was then rooted using root-to-tip regression. 50 trees of 50 tips each. To test the effect of data on the clock calibration, we randomly censored the dates of 5, 10, 25 and 50 tips, then reconstructed those from the clock.


\subsection{Data Collection} \label{subsec:dcollection}
For our second set of experiments we used a previously harvested set of data \citep{McCloskey14}. These data had been harvested from the HIV lanl database. Our third experiment also used data from Los Alamos Intra-Patient search interface \citep{LosAlamos}, except this time we used it to identify patients with both plasma and PBMC sequences from the ENV region of the HIV-1 genome. 

\subsection{Patient Data} \label{subsec:patdata}
 For our second set of experiments, we looked at RNA sequences from untreated patients from longitudinal studies with two or more clonal or single-genome (SGS) sequences available at each time point, with a known time- line relative to one of several reference points: HIV infection, seroconversion, presentation of symptomatic seroconversion illness, or birth. The seamples also wertr collected within 6 months (186 days) of this reference point, and where at least one of the subsequent (“follow-up”) time points occurred a minimum of 6 months after baseline. Known cases of superinfection were also excluded. 
 
We selected published studies that had well-characterized partial env sequences generated via clonal sequencing methods or single-genome amplification from HIV RNA in blood plasma or integrated DNA from PBMC cells. There was no condition for the time between sample collections. Fifteen individuals were included with serial samples from at least 3 time points (\ref{tab:patients}) (remove 3 from table). A total of X partial env sequences blood plasma specimens and Y partial env sequences from PBMC cells were used in our analysis.

\begin{table*}[!ht]\label{tab:patients} 
\begin{tabularx}{\textwidth}{| X | X | X | X | X | X | X | l |} 
\hline
Patient ID & \# of plasma samples & \# of PBMC samples & Total \# of Seqeunces & \# Plasma Time points & \# PBMC time points & Total Time points & Reference  \\ \hline
820 &       50 &       87 &      137 &        5 &       10 &       15 & Reference \\
821 &       76 &      192 &      268 &        7 &       17 &       17 & Reference  \\ 
822 &       32 &       98 &      130 &        3 &       10 &       10 & Reference  \\ 
824 &      100 &      107 &      207 &        7 &        9 &       13 & Reference  \\ 
10137 &       24 &      106 &      130 &        2 &       10 &       12 & Reference \\ 
10138 &       82 &      119 &      201 &        6 &       13 &       16 & Reference  \\
10586 &       16 &      121 &      137 &        2 &       12 &       14 & Reference \\ 
10769 &      229 &       96 &      325 &       10 &        4 &       11 & Reference \\ 
10770 &      190 &       31 &      221 &       11 &        2 &       11 & Reference \\ 
13889 &      151 &      132 &      283 &       13 &       14 &       18 & Reference \\ 
16616 &       16 &       95 &      111 &        2 &        5 &        5 & Reference \\ 
16617 &       16 &       89 &      105 &        2 &        5 &        5 & Reference  \\ 
16618 &       25 &       75 &      100 &        2 &        5 &        6 & Reference  \\ 
16619 &       13 &       60 &       73 &        2 &        6 &        6 & Reference \\ 
34375 &       16 &       73 &       89 &        1 &        7 &        7 & Reference \\ 
34382 &       37 &        5 &       42 &        3 &        1 &        4 & Reference \\ 
34391 &       13 &       38 &       51 &        3 &        5 &        6 & Reference  \\ 
34393 &       25 &       74 &       99 &        2 &        6 &        7 & Reference \\ 
34396 &       23 &       46 &       69 &        2 &        5 &        5 & Reference \\ 
34397 &       26 &       25 &       51 &        2 &        2 &        4 & Reference  \\ 
34399 &       61 &       88 &      149 &        5 &        9 &       12 & Reference \\ 
34400 &       54 &       23 &       77 &        4 &        1 &        5 & Reference \\ 
34405 &       27 &       29 &       56 &        3 &        2 &        4 & Reference \\ 
34408 &        5 &       73 &       78 &        2 &        8 &        8 & Reference\\ 
34410 &       35 &       60 &       95 &        2 &        6 &        6 & Reference\\ 
34411 &       25 &       43 &       68 &        3 &        3 &        6 & Reference   \\ \hline
\end{tabularx}

  \caption{Patient data collected from the HIV LANL database}
\end{table*}

\subsection{Sequence Aligment} \label{subsec:seqalign}
All of our sequences were annotated in FASTA formats, each containing multiple time points. Our simulated sequences were not simulated with any insertions or deletions, and no alignment was necessary, additionally, the dataset we used for our second set of experiments was already cleaned and aligned \citep{McCloskey14}. For our third dataset, we used the built in MUSCLE \citep{Muscle04} interface within AliView \citep{AliView14} to align the sequences, which were then visually inspected and modified. 

\subsection{Phylogeny Reconstruction} \label{subsec:phylo}
In all experiments all of our reconstructed phylogenies were maximum likelihood phylogenies reconstructed in RAxML \citep{Raxml14} using the GTR+$\Gamma$ model. Synthetic data were solely rooted with root-to-tip regression. For all other patient data, both outgroup rooting (against the HIV-1 B ancestor consensus \citep{HIVBANC}) and root-to-tip regression \cite{Ape04} were used to root the tree. We did not attempt to use BEAST \citep{BEAST} to build any phylogenies, as leaving dates unspecified on the amount of tips we had drastically increases the dimensionality of the problem, and would need a prohibitively large amount of samples from the MCMC chain to converge.

\subsection{Date Reconstruction} \label{subsec:daterecon}
Once the trees had been rooted, a general linear model with the normal family was constructed with the with expected number of substitutions as the input, and sampling time as the response. For all experiments using patient data, this fit was only over the plasma data, and the PBMCs expected subs were used as input. From this, we collected the difference between the observed sampling date, and the predicted date. 


\subsection{Patient Rejection} \label{subsec:hypot}
Patients whose linear model. Log ratio test. alpha value of 0.01\%. 
